react/features/polls/components/native/PollsPane.js

```diff
@@ -46,7 +46,7 @@ const PollsPane = (props: AbstractProps) => {
     return (
         <JitsiScreen
             contentContainerStyle = { chatStyles.pollPane }
-            disableForcedKeyboardDismiss = { !createMode }
+            disableForcedKeyboardDismiss = { true }
```

Comment:
Intentional change?

Reply:
Yes. With this set to true, scroll works better. 
It's the same issue that we used to have inside Settings screen.